scripts/install-multi-user.sh

```diff
@@ -700,6 +700,10 @@ EOF
 }
 
 welcome_to_nix() {
+    local -r NIX_UID_RANGES="${NIX_FIRST_BUILD_UID} ~ $((NIX_FIRST_BUILD_UID + NIX_USER_COUNT - 1))"
```

Comment:
I am not familiar with `~` for `[low..high)` ranges. Is it standard?

Reply:
I think it's [CJK specific](https://en.wikipedia.org/wiki/Tilde#Range). In this case, i used it to denote `[low..high]` range (i.e `{30001,30002, ... 30032}`). Should i change the format to something like `[30001..30032]`?

Reply:
Thanks for the background info! Yeah I think that might be better. `[...]` or `[..,)` is fine with me as I suppose we don't need to worry about the 0-build-user case :).

Reply:
alright, committed and updated screenshot.